ios/FluentUI.Demo/FluentUI.Demo/Info.plist

```diff
@@ -31,8 +31,6 @@
 	<true/>
 	<key>UILaunchStoryboardName</key>
 	<string>LaunchScreen</string>
-	<key>UIMainStoryboardFile</key>
-	<string>Main</string>
```

Comment:
should we also delete the storyboard file which declares a split view but we no longer implement split view delegate with your change?